audb/core/load.py

```diff
@@ -535,15 +535,27 @@ def _media(
 ) -> typing.Sequence[str]:
 
     if media is None:
-        media = db.files
-    elif isinstance(media, str):
+        return db.files
+    elif len(media) == 0:
+        return []
+
+    if isinstance(media, str):
         pattern = re.compile(media)
-        media = []
+        requested_media = []
         for m in db.files:
             if pattern.search(m):
-                media.append(m)
+                requested_media.append(m)
+        if len(requested_media) == 0:
+            raise ValueError(f"Could not find a media file matching '{media}'")
```

Comment:
In `load_media()` we show a slightly different error message:

```
f"Could not find '{media_file}' in {name} {version}"
```

Would it make sense to introduce a function to unify the error message? (same for the table case)

Reply:
I added a single function `_error_message_missing_object()` that now generates the error message at the six places we need it. The only downside is that the code is slightly harder to understand now.

Reply:
Great!